### PR TITLE
Board

### PR DESCRIPTION
**Pull request by Ju1111**
screen shot 2018-03-19 at 22 05 11

basics of placing two boards with 10x10 squares. no functionality

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Safisyx) [inline .eslintrc]: :+1:

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+  "extends": "react-app"
```
Comment (Safisyx) [inline .eslintrc:2]: :+1:

Comment (Ju1111) [reply .eslintrc:2]: trying to set up tests 😆

**Comment by Safisyx** [comment]
Looks wonderful, go Julia!!! :boom: :boat:

File: package.json [hunk 3]
```diff
@@ -15,5 +15,11 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
+  },
+  "devDependencies": {
```
Comment (Safisyx) [inline package.json:19]: Tests are one their way :stuck_out_tongue_winking_eye:

**Review by Safisyx — APPROVED** [review]
I think it's fine :+1: 
Please merge! :smile:

**Comment by Safisyx** [comment]
I'm in love with the colors of the squares :heart_eyes: :stuck_out_tongue_winking_eye:

**Pull request merged**